src/source/PeerConnection/SessionDescription.c

```diff
@@ -666,13 +666,15 @@ STATUS copyTransceiverWithCodec(PKvsPeerConnection pKvsPeerConnection, RTC_CODEC
     *pDidFindCodec = FALSE;
 
     CHK_STATUS(doubleListGetHeadNode(pKvsPeerConnection->pTransceievers, &pCurNode));
-    while (pCurNode != NULL && pTargetKvsRtpTransceiver == NULL) {
+    while (pCurNode != NULL) {
         CHK_STATUS(doubleListGetNodeData(pCurNode, &data));
-        pCurNode = pCurNode->pNext;
         pKvsRtpTransceiver = (PKvsRtpTransceiver) data;
         if (pKvsRtpTransceiver != NULL && pKvsRtpTransceiver->sender.track.codec == rtcCodec) {
             pTargetKvsRtpTransceiver = pKvsRtpTransceiver;
+            doubleListDeleteNode(pKvsPeerConnection->pTransceievers, pCurNode);
+            break;
```

Comment:
In general, we are trying to avoid break; and especially continue; control flow statements within loops. In case of simple loops. it's OK. However, consider a termination based on the condition check in the while loop like it was before. This is MINOR suggestion and can be safely ignored.

Reply:
i think this loop deserves the `simple loop` badge :)

Reply:
OK, you convince me